requirements/download-deps/bootstrap.txt

```diff
@@ -0,0 +1,3 @@
+setuptools==65.3.0
+wheel>=0.37.1
```

Comment:
Is there a reason we are not pinning the dependency on `wheel` here? I'd assume we would as it would guarantee the exact version that will end up in the lock file.